src/com/lftechnology/java/training/krishna/employeemanagement/controllers/LoginController.java

```diff
@@ -0,0 +1,474 @@
+	}
+
+	/**
+	 * This method is used to edit employee information
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @param id
+	 *            {@link int}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void editEmployee(Scanner scanner, int id) {
+
+		System.out.println("Do you want to edit your information? Please choose the options::\n");
+		String fullname = null;
+		String department = null;
+		String address = null;
+		String message = null;
+		int choice = 0;
+		employeeServiceImpl = new EmployeeServiceImpl();
+		editOptions(scanner);
+		while (true) {
+			choice = ValidationUtils.numberValidation(scanner);
+			if (choice == 1) {
+				System.out.println("Please enter the full name::");
+				fullname =
+					ValidationUtils.emptyValidation(scanner, ConstantUtils.FULLNAME);
+				break;
+			}
+			else if (choice == 2) {
+				System.out.println("Please enter the department::");
+				department =
+					ValidationUtils.emptyValidation(scanner, ConstantUtils.DEPARTMENT);
+				break;
+			}
+			else if (choice == 3) {
+				System.out.println("Please enter the address::");
+				address = ValidationUtils.emptyValidation(scanner, ConstantUtils.ADDRESS);
+				break;
+			}
+			else {
+				System.out.println(ConstantUtils.DESIRED_ACTION +"\n");
+				editOptions(scanner);
+			}
+		}
+		Employee emp =
+			employeeServiceImpl.updateEmployee(
+				id, fullname, department, address);
+		switch (choice) {
+		case 1:
+			message = ConstantUtils.FULLNAME;
+			break;
+		case 2:
+			message = ConstantUtils.DEPARTMENT;
+			break;
+		case 3:
+			message = ConstantUtils.ADDRESS;
+			break;
+		default:
+			message = "";
+			break;
+		}
+		if (emp.getId() > 0) {
+
+			System.out.println(message + " is updated successfully.\n");
+		}
+		else {
+			System.out.println("Failed to update " + message + ".\n");
+		}
+		showMainMenu(scanner, id);
+
+	}
+
+	/**
+	 * This method is used to display employee role
+	 *
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static void roleOptions() {
+
+		System.out.println("***********************************************"
+			+ "\n" + "**   1. Admin                                 **" + "\n"
+			+ "**   2. User                                  **" + "\n"
+			+ "************************************************" + "\n");
+	}
+
+	/**
+	 * This method is used to choose employee role
+	 *
+	 * @param scanner
+	 *            {@link Scanner}
+	 * @return role {@link String}
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	private static String chooseRole(Scanner scanner) {
+
+		String role = null;
+		int choice = 0;
+		roleOptions();
+		while (true) {
+			choice = ValidationUtils.numberValidation(scanner);
+			if (choice == 1) {
+				role = Role.Admin.toString();
+				break;
+			}
+			else if (choice == 2) {
+				role = Role.User.toString();
+				break;
+			}
+			else {
+				System.out.println(ConstantUtils.DESIRED_ACTION +"\n");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)